packages/api-server/src/middleware/withWebServer.ts

```diff
@@ -6,11 +6,21 @@ import type { Application } from 'express'
 
 import { getPaths } from '@redwoodjs/internal'
 
+type Contents = {
+  [content: string]: string
```

Comment:
```suggestion
  [path: string]: string
```

Reply:
I struggled with the name here, but path doesn't feel much better to me than content.  The values in the object that gets declared using this type are actually the html for the corresponding paths.  Does that impact what you think it should be called?